plugin.video.imdb.trailers/resources/lib/client.py

```diff
@@ -27,7 +27,7 @@
 CERT_FILE = TRANSLATEPATH('special://xbmc/system/certs/cacert.pem')
 
 
-def request(url, headers={}, timeout='20'):
+def request(url, headers={}, post=None, timeout='20'):
```

Comment:
A mutable default argument is a bad practice.

Reply:
changed it to None. Thanks